### DISCUSSION TITLE
Test - Train Splitting

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#783] [Q&A] [answered]
Selamat sore, saya mau bertanya y = banks['y'] X = banks.drop(['y'], axis=1), untuk syntax ini di python apakah harus melakukan drop yaa? bukankah kalo drop ini misalkan kita menghap
<img width="1440" alt="Screen Shot 2023-10-16 at 16 33 51" src="https://github.com/teamalgoritma/community/assets/123171973/d9e86898-0f21-4ad8-8d33-747eb50674cf">
us Kolom yang ada?

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Pak @mosesnugraha , 

Berdasarkan error yang terlampir, sepertinya data Bapak tidak memiliki kolom `y`. 
Bisa dilihat kembali nama kolom yang ingin diakses dan mengganti `y` dengan nama kolom yang sebenarnya.

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma
